src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/Validation/ModelValidationContext.cs

```diff
@@ -22,10 +25,5 @@ public class ModelValidationContext
         /// Gets or sets the model container object.
         /// </summary>
         public object Container { get; set; }
```

Comment:
Why is this property not like the others? As I said earlier, make all of the properties get-only and don't expose a bizarre mix in this class if possible. Note `Container` is never set except in init syntax -- at least as shown in this PR.